src/lib.rs

```diff
@@ -97,14 +97,9 @@ impl StarReference {
 #[derive(Clone)]
 pub struct StarSettings {
     reference_path: String,
-    multn: isize,
     args: Vec<String>,
 }
 
-/// Set the --outSAMmultNmax parameter of STAR.
-/// Max number of multiple alignments for a read that will be output to the SAM/BAM files.
-const DEFAULT_MULTN: isize = -1;
```

Comment:
Is -1 already the default and hence we don't need to do this?

Reply:
Sorry, yes. Mean to add a comment to that effect. I'll add it to the PR description above.

Reply:
See https://github.com/10XGenomics/STAR/blob/e213d26f068af6f2b9997446b60341ebdeb866bc/source/parametersDefault#L313-L315